packages/rancher-alerting-drivers/charts/templates/cluster-role.yaml

```diff
@@ -0,0 +1,66 @@
+{{- $admin := "alerting-drivers-admin" -}}
+{{- $edit  := "alerting-drivers-edit"  -}}
+{{- $view  := "alerting-drivers-view"  -}}
```

Comment:
No need to make these Helm variables, let's just add them directly under names since they are only used once

Reply:
1. They are referred at 2 places so I made them variables. 
2. They use the app's full name now.